google-marketplace/README.md

```diff
@@ -0,0 +1,173 @@
+# Google Cloud Launcher
+
+## Overview
+
+Datadog provides infrastructure monitoring, application performance monitoring, and log management in a single-pane-of-glass view so teams can scale rapidly and maintain operational excellence.
+
+Installing the Datadog package via Google Cloud Launcher deploys the Datadog agent on every node in your Kubernetes cluster, and configures it with a secure, RBAC-based authentication and authorization model.
+
+## Installation
+
+### Quick install with Google Cloud Marketplace
+
+Get up and running with a few clicks! Install the Datadog agent daemonset to a
+Google Kubernetes Engine cluster using Google Cloud Marketplace.
+
+Prior to that:
+
+- Create a Datadog [account](https://www.datadoghq.com/)
+- Get your Datadog API key [here](https://app.datadoghq.com/account/settings#api)
+
+Then follow the [on-screen instructions](https://console.cloud.google.com/marketplace/details/datadog-saas/datadog).
+
+### Command line instructions
+
+Follow these instructions to install the Datadog agent from the command line.
+
+#### Prerequisites (one time setup)
+
+##### Command-line tools
+
+You'll need the following tools in your development environment:
+
+- [gcloud](https://cloud.google.com/sdk/gcloud/)
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+##### Create a Datadog account
+
+- Create a Datadog [account](https://www.datadoghq.com/)
+- Get your Datadog API key [here](https://app.datadoghq.com/account/settings#api)
+
+##### Create a Google Kubernetes Engine cluster
+
+Create a new cluster from the command line:
+
+```shell
+export CLUSTER=datadog-cluster
+export ZONE=us-west1-a
+
+gcloud container clusters create "$CLUSTER" --zone "$ZONE"
+```
+
+Configure `kubectl` to connect to the new cluster:
+
+```shell
+gcloud container clusters get-credentials "$CLUSTER" --zone "$ZONE"
+```
+
+###### Clone this repository
+
+Clone this repository.
+
+```shell
+git clone git@github.com:DataDog/datadog-agent.git
+```
+
+###### Install the Application resource definition
+
+An Application resource is a collection of individual Kubernetes components,
+such as Services, Deployments, and so on, that you can manage as a group.
+
+To set up your cluster to understand Application resources, navigate to the
+`google-marketplace` folder in the repository, and run the following command:
+
+```shell
+make crd/install
+```
+
+You need to run this command once.
+
+The Application resource is defined by the
+[Kubernetes SIG-apps](https://github.com/kubernetes/community/tree/master/sig-apps)
+community. The source code can be found on
+[github.com/kubernetes-sigs/application](https://github.com/kubernetes-sigs/application).
+
+#### Install the Application
+
+Navigate to the `google-marketplace` folder:
+
+```shell
+cd datadog-agent/google-marketplace
+```
+
+##### Configure the app with environment variables
```

Comment:
Is `app` a real concept name? Or just a shortcut for `application`? If so, we should use `application` instead of `app`